### ISSUE TITLE
Import fails if asset on target system is deleted

### ISSUE THREAD

**Issue by gglnx** [verbb/zen#9]
### Describe the bug

If an asset exists on both systems, but was deleted on the target system, the import fails:

```
Unable to open [deleted_asset].jpg.
/vendor/craftcms/cms/src/fs/Local.php:354
/vendor/craftcms/cms/src/models/Volume.php:465
```

My exception were that the asset and its file should be restored.

### Steps to reproduce

1. Generate an export
2. Import it on another system
3. Delete there a file
4. Try again to import

### Craft CMS version

4.8.2

### Plugin version

1.0.3@dev

### Multi-site?

Yes

**Comment by engram-design** [comment]
So restoring assets is a tricky thing, because I believe the files themselves are hard-deleted. So you mention the asset exists on both installs, it's not deleted on one install? Or the asset on the target system is soft-deleted, and the asset file is hard-deleted?

**Comment by gglnx** [comment]
I deleted the asset on the target system in the control panel. Craft soft-deletes the element, but hard-deletes the file on the filesystem.

**Comment by engram-design** [comment]
Fixed for the next release. To get this early, run `composer require verbb/zen:"dev-craft-4 as 1.0.3"`.

**Comment by engram-design** [comment]
Fixed in [1.0.4](https://github.com/verbb/zen/releases/tag/1.0.4)
